ckanext/datagovtheme/helpers.py

```diff
@@ -780,13 +781,17 @@ def get_bureau_info(bureau_code):
             resource = urllib.request.urlopen(url, timeout=sec_timeout)
         except Exception:
             file_obj = open(filename)
-            file_conent = file_obj.read()
+            file_content = file_obj.read()
             # touch the file, so that it wont keep re-trying and slow down page loading
             os.utime(filename, None)
         else:
             file_obj = open(filename, 'w+')
-            file_conent = resource.read()
-            file_obj.write(file_conent)
+            file_content = resource.read()
+            # Handle differences between py2 and py3. Should use else statement for py3.
+            if isinstance(file_content, bytes):
```

Comment:
This is a little silly because you're  reading the file as binary, then converting text. Just read the file as text from the start. Check the docs for open(), you can specify your encoding there in order  to get string data.